### ISSUE TITLE
Search: add role

### ISSUE THREAD

**Issue by mattsharkey** [web-illinois/toolkit#233]
The site search form should have a role of search. This can go on the <form> element or its immediate container.

**Comment by bryanjonker-illinois** [comment]
From Todd Nelson -- https://teams.microsoft.com/l/message/19:734981213bce44f2a0f31e98a560fe08@thread.tacv2/1647448476218?tenantId=44467e6f-462c-4ea2-823f-7800de5434e3&groupId=7ecdbcb2-4a6c-438d-828c-70287b84f487&parentMessageId=1647448476218&teamName=Website%20Implementation%20Guidelines%20Group&channelName=Web%20Components&createdTime=1647448476218

Hi Web Components folks! I’ve implemented several web components on https://www.itproforum.illinois.edu and I’ve received some comments from Jon Gunderson about il-search:

“The button with the eyeglass should move the aria-label from the SVG element to the button element… The role=image on the SVG element is not valid because ARIA does not define a role IMAGE, only the role IMG. The SVG element should have role aria-hidden, so no information on the svg image is communicated through the accessibility API.”
